.circleci/config.yml

```diff
@@ -313,9 +347,12 @@ workflows:
           filters:
               branches:
                 only: master
-      - version_and_publish:
+      - rename_latest_to_alpha:
```

Comment:
I'm assuming we'll remove this in a following PR?

Reply:
Yup